app/controllers/transfer_packages_controller.rb

```diff
@@ -17,16 +72,54 @@ def find_sample
 
   private
 
+  def transfer_package_params
+    params.require(:transfer_package).permit(
+      :receiver_institution,
+      :recipient,
+      :includes_qc_info,
+    ).tap do |whitelisted|
+      if sample_transfers_attributes = params[:transfer_package][:sample_transfers_attributes]
+        sample_transfers_attributes.reject! { |_, attributes| attributes[:sample_uuid].blank? }
+        sample_transfers_attributes.transform_values! { |attributes|
+          ActionController::Parameters.new(attributes).permit(
+            :sample_uuid
+          )
+        }
+        if Rails::VERSION::MAJOR >= 5
+          whitelisted[:sample_transfers_attributes] = sample_transfers_attributes.permit!
+        else
+          whitelisted[:sample_transfers_attributes] = sample_transfers_attributes
+        end
+      else
+        whitelisted[:sample_transfers_attributes] = {}
+      end
+    end
+  end
+
+  def available_institutions
+    Institution.where.not(id: @transfer_package.sender_institution.id)
+  end
```

Comment:
This could be a helper method, so you wouldn't need to instantiate an `@available_institutions` ivar. Just add `helper_method :available_institutions` to the controller.

Reply:
In that case I just memoize the ivar :)